lib/MaxMind/DB/Writer/Tree.pm

```diff
@@ -263,13 +305,18 @@ sub write_tree {
 
 sub new_from_frozen_tree {
     my $class = shift;
-    my ( $filename, $callback, $database_type, $description )
+    my (
+        $filename,                $callback, $database_type, $description,
+        $merge_record_collisions, $merge_strategy
+        )
         = validated_list(
         \@_,
-        filename              => { isa => 'Str' },
-        map_key_type_callback => { isa => 'CodeRef' },
-        database_type         => { isa => 'Str', optional => 1 },
-        description           => { isa => 'HashRef[Str]', optional => 1 },
+        filename                => { isa => 'Str' },
+        map_key_type_callback   => { isa => 'CodeRef' },
+        database_type           => { isa => 'Str', optional => 1 },
+        description             => { isa => 'HashRef[Str]', optional => 1 },
+        merge_record_collisions => { isa => 'Bool', optional => 1 },
+        merge_strategy => { isa => $MergeStrategyEnum, optional => 1 }
```

Comment:
`tidyall` disagrees. I think it is like that as it doesn't want to wrap the hashref.